veridian-slang/build.rs

```diff
@@ -46,21 +91,22 @@ fn main() {
         .generate()
         .expect("Unable to generate bindings");
 
+    println!("cargo:rustc-link-search=native={}", slang_install.display());
     println!(
-        "cargo:rustc-link-search=native={}/slang_wrapper/slang/lib",
-        env::var("CARGO_MANIFEST_DIR").unwrap()
+        "cargo:rustc-link-search=native={}",
+        wrapper_install.display()
     );
     // println!("cargo:rustc-link-search=native=/usr/lib");
 
     println!("cargo:rustc-link-lib=static=slangwrapper");
-    println!("cargo:rustc-link-lib=static=slangcompiler");
-    println!("cargo:rustc-link-lib=static=slangruntime");
-    println!("cargo:rustc-link-lib=static=slangparser");
-    println!("cargo:rustc-link-lib=static=slangcore");
+    println!("cargo:rustc-link-lib={link_type}=svlang");
+    println!("cargo:rustc-link-lib={link_type}=fmt");
```

Comment:
```suggestion
    println!("cargo:rustc-link-lib=fmt");
```
Removing this link type from this and the following line allows rustc to link to the dynamic libraries for fmt and mimalloc.
```
❯ ldd target/debug/veridian
	linux-vdso.so.1 (0x000075bf62602000)
	libfmt.so.10 => /usr/lib/libfmt.so.10 (0x000075bf625a9000)
	libmimalloc.so.2 => /usr/lib/libmimalloc.so.2 (0x000075bf62557000)
	libstdc++.so.6 => /usr/lib/libstdc++.so.6 (0x000075bf5f600000)
	libgcc_s.so.1 => /usr/lib/libgcc_s.so.1 (0x000075bf6252a000)
	libm.so.6 => /usr/lib/libm.so.6 (0x000075bf6243f000)
	libc.so.6 => /usr/lib/libc.so.6 (0x000075bf5f414000)
	/lib64/ld-linux-x86-64.so.2 => /usr/lib64/ld-linux-x86-64.so.2 (0x000075bf62604000)
```

Reply:
Done.